src/rdkafka_mock_handlers.c

```diff
@@ -146,13 +150,28 @@ static int rd_kafka_mock_handle_Produce(rd_kafka_mock_connection_t *mconn,
                                             resp, mpart->start_offset);
                                 }
                         }
+
+                        if (rkbuf->rkbuf_reqhdr.ApiVersion >= 8) {
+                                /* TODO: Add support for injecting RecordErrors
+                                 * 0 record errors for now */
+                                rd_kafka_buf_write_arraycnt(resp, 0);
+
+                                /* error_message */
+                                rd_kafka_buf_write_str(resp, NULL, 0);
+
+                                rd_kafka_buf_write_tags_empty(resp);
+                        }
                 }
+
+                /* Topic Tags */
+                rd_kafka_buf_write_tags_empty(resp);
         }
 
         if (rkbuf->rkbuf_reqhdr.ApiVersion >= 1) {
                 /* Response: ThrottleTime */
                 rd_kafka_buf_write_i32(resp, 0);
         }
+        rd_kafka_buf_write_tags_empty(resp);
 
         rd_kafka_mock_connection_send_response(mconn, resp);
```

Comment:
If writing tags at line 174, use the new function to avoid writing tags twice
```suggestion
        rd_kafka_mock_connection_send_response0(mconn, resp, rd_true);
```